README.md

```diff
@@ -1,8 +1,8 @@
 # airflow-fivetran-dbt
-Example orchestration pipeline for Fivetran + dbt managed by Airflow
+The purpose of this github repo is to provide an example how an orchestration pipeline for Fivetran + dbt managed by Airflow would look like. If you have any questions about this, feel free to ask in our [dbt Slack community](https://community.getdbt.com/).
```

Comment:
Can we replace the added text with "The purpose of this github repo is to provide an example of an orchestration pipeline for Fivetran + dbt managed by Airflow. If you have any questions about this, feel free to ask in our [dbt Slack community](https://community.getdbt.com/)."